types/office-js-preview/index.d.ts

```diff
@@ -137233,6 +137377,14 @@ declare namespace PowerPoint {
               Specifying `$all` for the load options loads all the scalar properties (such as `Range.address`) but not the navigational properties (such as `Range.format.fill.color`).
              */
             $all?: boolean;
+            /**
+             * Specifies whether the font uses all caps, where lowercase letters are shown as capital letters. Returns `null` if the `TextRange` contains text fragments both with and without all caps.
```

Comment:
There is a problem with *all* of our *LoadOptions reference topics. The problem would be made worse if we change these descriptions in ShareFontLoadOptions in the way I recommended for ShapeOptions, so I think we should just leave the property descriptions in this interface the way they are for now.

Reply:
I think the descriptions for these types of objects (*LoadOptions, *UpdateData, *Data, etc.) are generated by copying the descriptions from the main object. For this to be fixed, you'll need to discuss with Jonah and the other infra folks